usaspending_api/database_scripts/job_archive/management/commands/generate_cares_act_test_data_helpers.py

```diff
@@ -0,0 +1,265 @@
+"""
+Jira Ticket Number(s): DEV-5343
+
+    Helper functions/classes for CARES Act test data generation.
+
+Expected CLI:
+
+    None.  This is a helper file, silly.
+
+Purpose:
+
+    This script generates or participates in the generation of sample CARES Act data for testing
+    and development purposes.  It generates these data from existing data by duplicating and
+    modifying existing submissions and File A/B/C records.  Data points are adjusted in an attempt
+    to make them seem realistic and true to their actual source submissions.
+
+    These data will not be perfect, obviously, but they should be sufficient for testing.
+
+Life expectancy:
+
+    This file should live until CARES Act features have gone live.
+
+    Be sure to delete all files/directories associated with this ticket:
+        - job_archive/management/commands/generate_cares_act_test_data_helpers.py
+        - job_archive/management/commands/generate_cares_act_test_def_codes.py
+        - job_archive/management/commands/generate_cares_act_test_monthly_submissions.py
+        - job_archive/management/commands/generate_cares_act_test_data_sqls
+
+"""
+import logging
+import re
+
+from argparse import ArgumentTypeError
+from collections import namedtuple
+from pathlib import Path
+from usaspending_api.common.helpers.sql_helpers import execute_sql_return_single_value, execute_dml_sql
+from usaspending_api.common.helpers.timing_helpers import ScriptTimer
+from usaspending_api.submissions.models import SubmissionAttributes
+from datetime import timedelta, date
+
+
+logger = logging.getLogger("script")
+
+
+class OneLineTimer(ScriptTimer):
+    def log_starting_message(self):
+        pass
+
+    def log_success_message(self):
+        pass
+
+
+class DEV5343Mixin:
+
+    fiscal_year = None
+    fiscal_quarter = None
+    fiscal_period = None
+    allow_rds = False
+    vacuum = False
+
+    clone_periods = None
+    period_ratios = None
+    period_starts = None
+    period_ends = None
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            "fiscal year", type=self.validate_fiscal_year, help='Fiscal year to be "enhanced".',
```

Comment:
```suggestion
            "fiscal-year", type=self.validate_fiscal_year, help='Fiscal year to be "enhanced".',
```
do commands support spaces in flags?

Reply:
Yes.  They're just for display purposes.

Reply:
To expand, these are not flags/switches.  They're fixed position arguments.